### ISSUE TITLE
Problem with authentication in shipit/frontend

### ISSUE THREAD

**Issue by garbas** [mozilla/release-services#1633]
when clicking any of the buttons when doing release promotion work it throws the authentication error.

**Comment by rail** [comment]
The "auth failure" message is a bit misleading, it's a kind of catch-all message in this case.

I believe this is cased by #1650. At the end of operations we send an IRC message, but fail to do so:
```
Oct 26 10:03:03 104.196.232.28 mozilla/release-services/production/shipit/api: 2018-10-26 17:03:03.015198 flask.app: Exception on /releases/Fennec-64.0b4-build1 [DELETE] Traceback (most recent call last):   File "/nix/store/gmq8dvlikhj3c689wsjlinhj3p3rijxc-python3.6-Flask-1.0.2/lib/python3.6/site-packages/flask/app.py", line 2292, in wsgi_app     response = self.full_dispatch_request()   File "/nix/store/gmq8dvlikhj3c689wsjlinhj3p3rijxc-python3.6-Flask-1.0.2/lib/python3.6/site-packages/flask/app.py", line 1815, in full_dispatch_request     rv = self.handle_user_exception(e)   File "/nix/store/xl3q9wk4ggmpzvgd9r5i82lj9kms8rr7-python3.6-Flask-Cors-3.0.6/lib/python3.6/site-packages/flask_cors/extension.py", line 161, in wrapped_function     return cors_after_request(app.make_response(f(*args, **kwargs)))   File "/nix/store/gmq8dvlikhj3c689wsjlinhj3p3rijxc-python3.6-Flask-1.0.2/lib/python3.6/site-packages/flask/app.py", line 1718, in handle_user_exception     reraise(exc_type, exc_value, tb)   File "/nix/store/gmq8dvlikhj3c689wsjlinhj3p3rijxc-python3.6-Flask-1.0.2/lib/python3.6/site-packages/flask/_compat.py", line 35, in reraise     raise value   File "/nix/store/gmq8dvlikhj3c689wsjlinhj3p3rijxc-python3.6-Flask-1.0.2/lib/python3.6/site-packages/flask/app.py", line 1813, in full_dispatch_request     rv = self.dispatch_request()   File "/nix/store/gmq8dvlikhj3c689wsjlinhj3p3rijxc-python3.6-Flask-1.0.2/lib/python3.6/site-packages/flask/app.py", line 1799, in dispatch_request     return self.view_functions[rule.endpoint](**req.view_args)   File "/nix/store/a67dj450kxhggcyrm5c9w04wqv8wx5x4-python3.6-python3.6-connexion-1.5.3/lib/python3.6/site-packages/connexion/decorators/decorator.py", line 73, in wrapper     response = function(request)   File "/nix/store/a67dj450kxhggcyrm5c9w04wqv8wx5x4-python3.6-python3.6-connexion-1.5.3/lib/python3.6/site-packages/connexion/decorators/uri_parsing.py", line 117, in wrapper     response = function(request)   File "/nix/store/a67dj450kxhggcyrm5c9w04wqv8wx5x4-python3.6-python3.6-connexion-1.5.3/lib/python3.6/site-packages/connexion/decorators/validation.py", line 336, in wrapper     return function(request)   File "/nix/store/a67dj450kxhggcyrm5c9w04wqv8wx5x4-python3.6-python3.6-connexion-1.5.3/lib/python3.6/site-packages/connexion/decorators/response.py", line 104, in wrapper     response = function(request)   File "/nix/store/a67dj450kxhggcyrm5c9w04wqv8wx5x4-python3.6-python3.6-connexion-1.5.3/lib/python3.6/site-packages/connexion/decorators/decorator.py", line 44, in wrapper     response = function(request)   File "/nix/store/a67dj450kxhggcyrm5c9w04wqv8wx5x4-python3.6-python3.6-connexion-1.5.3/lib/python3.6/site-packages/connexion/decorators/parameter.py", line 207, in wrapper     return function(**kwargs)   File "/nix/store/5hrl1spgh1jc0z64riz1x3h6x1jd9jkg-python3.6-python3.6-mozilla-backend-common-1.0.0/lib/python3.6/site-packages/backend_common/auth0.py", line 133, in decorated     return view_func(*args, **kwargs)   File "/nix/store/c6zsr7fx348mn2m955qv9zbk118sybmj-python3.6-mozilla-shipit-api-1.0.0/lib/python3.6/site-packages/shipit_api/api.py", line 67, in decorated     return view_func(*args, **kwargs)   File "/nix/store/c6zsr7fx348mn2m955qv9zbk118sybmj-python3.6-mozilla-shipit-api-1.0.0/lib/python3.6/site-packages/shipit_api/api.py", line 214, in abandon_release     notify_via_irc(f'Release {r.product} {r.version} build{r.build_number} was just canceled.')   File "/nix/store/c6zsr7fx348mn2m955qv9zbk118sybmj-python3.6-mozilla-shipit-api-1.0.0/lib/python3.6/site-packages/shipit_api/api.py", line 42, in notify_via_irc     'message': f'{owners}: {message}',   File "/nix/store/jka9y48bv8d40js9i00jsrg0b51slq59-python3.6-python3.6-taskcluster-4.0.1/lib/python3.6/site-packages/taskcluster/notify.py", line 87, in irc     return self._makeApiCall(self.funcinfo["irc"], *args, **kwargs)   File "/nix/store/jka9y48bv8d40js9i00jsrg0b51slq59-python3.6-python3.6-taskcluster-4.0.1/lib/python3.6/site-packages/taskcluster/client.py", line 262, in _makeApiCall     response = self._makeHttpRequest(entry['method'], _route, payload)   File "/nix/store/jka9y48bv8d40js9i00jsrg0b51slq59-python3.6-python3.6-taskcluster-4.0.1/lib/python3.6/site-packages/taskcluster/client.py", line 534, in _makeHttpRequest     superExc=None taskcluster.exceptions.TaskclusterRestFailure: You do not have sufficient scopes. You are missing the following scopes:  ``` notify:irc-channel:#releaseduty ```  You have the scopes:  ``` [   "assume:project:shipit:bot",   "project:shipit:analysis/manage",   "project:shipit:bot",   "queue:get-artifact:project/gecko/android-*",   "secrets:get:project/shipit/bot/*" ] ```  This request requires you to satisfy this scope expression:  ``` notify:irc-channel:#releaseduty ```  ---  * method:     irc * errorCode:  InsufficientScopes * statusCode: 403 * time:       2018-10-26T17:03:02.875Z 
```

**Comment by rail** [comment]
Should be fixed by #1659
